# Translations template for vigilo-correlator.
# Copyright (C) 2010 ORGANIZATION
# This file is distributed under the same license as the vigilo-correlator
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: vigilo-correlator 2.0.0\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2010-11-04 10:51+0100\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/correlator/compute_hls_states.py:148
#, python-format
msgid "Dependancies have the following weights: %r"
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:186
#, python-format
msgid ""
"High-level service %(name)r: weight = %(weight)r, state = %(state)r "
"(%(output)r)"
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:238
#, python-format
msgid "End of impact for HLS %r"
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:255
#, python-format
msgid "End of impact for item (%(hostname)r, %(servicename)r)"
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:277
#, python-format
msgid "HLS %(hls1)r is impacted by HLS %(hls2)r"
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:295
#, python-format
msgid ""
"The HLS %r already belonged to this impacted path, ignoring request to add it"
" again."
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:324
msgid "Copying path"
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:339
#, python-format
msgid "HLS %(hls)r is impacted by item (%(host)r, %(service)r)"
msgstr ""

#: src/vigilo/correlator/compute_hls_states.py:377
#, python-format
msgid "Setting weight for '%s' to None"
msgstr ""

#: src/vigilo/correlator/correvent.py:111
#, python-format
msgid "Updating existing correlated event (%r)"
msgstr ""

#: src/vigilo/correlator/correvent.py:114
#, python-format
msgid "Got a reference to a non-existent correlated event (%r), adding as new"
msgstr ""

#: src/vigilo/correlator/correvent.py:124
#, python-format
msgid "Raw event ignored. Reason: status = %r"
msgstr ""

#: src/vigilo/correlator/correvent.py:150
#, python-format
msgid ""
"Got a reference to a nonexistent correlated event (%r), skipping this "
"aggregate"
msgstr ""

#: src/vigilo/correlator/correvent.py:193
msgid "Creating new correlated event"
msgstr ""

#: src/vigilo/correlator/correvent.py:308
msgid "Sending correlated event to syslog"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:44
#, python-format
msgid "Received request to add an event on HLS \"%s\""
msgstr ""

#: src/vigilo/correlator/db_insertion.py:52
#, python-format
msgid ""
"Got a reference to a non configured item (%(host)r, %(service)r), skipping "
"event"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:85
#, python-format
msgid "Updating event %r"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:91
#, python-format
msgid ""
"Ignoring request to create a new event with state \"%s\" (nothing alarming "
"here)"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:99
msgid "Creating new event"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:102
msgid "Multiple matching events found, skipping."
msgstr ""

#: src/vigilo/correlator/db_insertion.py:128
#: src/vigilo/correlator/db_insertion.py:163
#: src/vigilo/correlator/db_insertion.py:207
#: src/vigilo/correlator/db_insertion.py:227
msgid "Got exception"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:148
#, python-format
msgid "Got a reference to a non configured high-level service (%(service)r)"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:181
#, python-format
msgid ""
"Got a reference to a non configured item (%(host)r, %(service)r), skipping "
"state"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:244
#, python-format
msgid "Merging aggregate #%(src)d with aggregate #%(dest)d"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:255
msgid "merge_aggregates: Got a reference to a nonexistent source aggregate, aborting"
msgstr ""

#: src/vigilo/correlator/db_insertion.py:265
#, python-format
msgid ""
"merge_aggregates: Got a reference to a nonexistent destination aggregate %r, "
"aborting"
msgstr ""

#: src/vigilo/correlator/handle_ticket.py:32
#, python-format
msgid ""
"handle_ticket: Trouble ticket message received. Timestamp = %(timestamp)r. "
"Impacted HLS = %(hls)r. Ticket id = %(ticket_id)r. acknowledgement_status = "
"%(ack_status)r. Message = %(message)r."
msgstr ""

#: src/vigilo/correlator/handle_ticket.py:54
#, python-format
msgid "handle_ticket: No matching trouble ticket found : %r"
msgstr ""

#: src/vigilo/correlator/handle_ticket.py:60
#, python-format
msgid "handle_ticket: Several events seem to be associated with this ticket : %r"
msgstr ""

#: src/vigilo/correlator/handle_ticket.py:63
#, python-format
msgid ""
"handle_ticket: The event %(event_id)r is associated with the given ticket "
"(%(ticket_id)r)"
msgstr ""

#: src/vigilo/correlator/handle_ticket.py:85
#, python-format
msgid "handle_ticket: Got exception while updating event %r history"
msgstr ""

#: src/vigilo/correlator/handle_ticket.py:89
#, python-format
msgid "handle_ticket: Event %r history updated successfully."
msgstr ""

#: src/vigilo/correlator/memcached_connection.py:144
#, python-format
msgid "Deleted %(nb_deleted)d expired correlation contexts"
msgstr ""

#: src/vigilo/correlator/memcached_connection.py:187
#, python-format
msgid "Establishing connection to MemcacheD server (%s)..."
msgstr ""

#: src/vigilo/correlator/memcached_connection.py:209
#, python-format
msgid "Trying to set value '%(value)s' for key '%(key)s'."
msgstr ""

#: src/vigilo/correlator/memcached_connection.py:254
#, python-format
msgid "Trying to get the value of the key '%(key)s'."
msgstr ""

#: src/vigilo/correlator/memcached_connection.py:303
#, python-format
msgid "Trying to delete the key '%(key)s'."
msgstr ""

#: src/vigilo/correlator/registry.py:55
#, python-format
msgid "Rule %r has already been registered, ignoring attempt to re-register it."
msgstr ""

#: src/vigilo/correlator/registry.py:63
#, python-format
msgid "The rule %(depended)r must be loaded before %(dependent)r."
msgstr ""

#: src/vigilo/correlator/registry.py:71
#, python-format
msgid "Successfully registered rule %r"
msgstr ""

#: src/vigilo/correlator/registry.py:129
msgid ""
"Singleton has already been instanciated, ignoring attempt to create a new "
"instance."
msgstr ""

#: src/vigilo/correlator/registry.py:159
#, python-format
msgid "Not a valid rule name \"%s\""
msgstr ""

#: src/vigilo/correlator/registry.py:165
#, python-format
msgid "Unable to load rule \"%s\""
msgstr ""

#: src/vigilo/correlator/twisted_service.py:50
msgid "The topology has been reloaded."
msgstr ""

#: src/vigilo/correlator/twisted_service.py:64
msgid ""
"Could not set signal handlers. The correlator may not be able to shutdown "
"cleanly"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:113
msgid ""
"No information from Vigiconf concerning the topology's last modification "
"date. Therefore the topology has NOT been rebuilt."
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:125
msgid "Topology has been reloaded."
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:151
#, python-format
msgid "Sending this payload to the XMPP bus: %r"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:266
#, python-format
msgid "Got item: %s"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:271
#, python-format
msgid "Skipping unrecognized item (%s)"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:280
msgid "The correlator is shutting down due to a previous error"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:333
#, python-format
msgid "Rule %(rule_name)s timed out"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:449
#, python-format
msgid "Spawning for payload: %s"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:456
msgid "Received invalid XMPP item ID (None)"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:466
#, python-format
msgid "Validation failure: %s"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:506
msgid "Could not connect to the database server, make sure it is running"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:614
#, python-format
msgid "Chat message to forward: '%s'"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:625
msgid "Connection initialized"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:635
#, python-format
msgid "Starting the queue manager with a delay of %f seconds."
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:649
msgid "Connection lost"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:687
#, python-format
msgid ""
"Unable to forward one-to-one message to XMPP server (no connection "
"established). The message (%s) has been stored for later retransmission."
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:706
#, python-format
msgid ""
"Unable to forward the message (%(error)r), it has been stored for later "
"retransmission (%(xml_src)s)"
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:717
#, python-format
msgid "No destination node configured for messages of type '%s'. Skipping."
msgstr ""

#: src/vigilo/correlator/actors/rule_dispatcher.py:726
#, python-format
msgid ""
"Message from Socket impossible to forward (no connection to XMPP server), the"
" message is stored for later reemission (%s)"
msgstr ""

#: src/vigilo/correlator/actors/rule_runner.py:56
#, python-format
msgid "Rule runner: process begins for rule \"%s\""
msgstr ""

#: src/vigilo/correlator/actors/rule_runner.py:57
#, python-format
msgid "Process id: %(pid)r | Parent id: %(ppid)r | Rule name: %(name)s"
msgstr ""

#: src/vigilo/correlator/actors/rule_runner.py:65
#, python-format
msgid "Rule runner: process ends for rule \"%s\""
msgstr ""

#: src/vigilo/correlator/rules/adapt_frequency.py:88
#, python-format
msgid "Ignoring non-existent low-level service (%(service)s on %(host)s)"
msgstr ""

#: src/vigilo/correlator/rules/adapt_frequency.py:153
#, python-format
msgid "Asking Nagios to set CHECK_INTERVAL for %(object)r to %(delay)d periods."
msgstr ""

#: src/vigilo/correlator/rules/example.py:23
#, python-format
msgid "id %(id)s payload %(payload)s"
msgstr ""

#: src/vigilo/correlator/rules/hls_deps.py:98
#, python-format
msgid "Impacted HLS = %(affected_hls)r for (%(hostname)r, %(servicename)r)"
msgstr ""

#: src/vigilo/correlator/rules/priority.py:73
#, python-format
msgid "The priority for the alert on (%(hostname)r, %(servicename)r) is %(priority)r."
msgstr ""

#: src/vigilo/correlator/rules/priority.py:114
#, python-format
msgid ""
"The priority of the correlated event(s) %(updated)d has been updated, based "
"on information about the HLS %(hls)r"
msgstr ""

#: src/vigilo/correlator/rules/update_occurrences_count.py:85
#, python-format
msgid ""
"The state for '%(servicename)s' on '%(hostname)s' did not change or indicates"
" no problem whatsoever. The occurrences counter has not been incremented. New"
" state = %(statename)r vs %(previous_statename)r -- Occurrences: "
"%(occurrences)d"
msgstr ""

#: src/vigilo/correlator/rules/update_occurrences_count.py:102
#, python-format
msgid "Host: %(hostname)s, service: %(servicename)s, occurrences: %(occurrences)d"
msgstr ""

